components/jwt-sidecar/get-keycloak-identities.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash -x
+
+IDSOUTDIR="/usr/share/secrets/tsi-secrets/identities"
+JWTFILE="/jwt/token"
+# IDSREQFILE - Identities request file from Pod Annotation
+IDSREQFILE="/pod-metadata/tsi-identities"
+
+# when we decide to pass the Keycloak address on cluster level:
+# validate if KEYCLOAK_ADDR env. variable is set
+# if [ "$KEYCLOAK_ADDR" == "" ]; then
+#   echo "KEYCLOAK_ADDR must be set"
+#   exit 1
+# fi
+
+# make sure that JWT file exists
+if [ ! -s "$JWTFILE" ]; then
+   echo "$JWTFILE does not exist. Make sure Trusted Identity is setup correctly"
+   exit 1
+fi
+
+# since annotations are provided in YAML format,
+convert YAML to JSON for easier manipulations
+if [ ! -s "$IDSREQFILE" ]; then
+   echo "$IDSREQFILE contains no data. Nothing to do"
+   exit 1
+fi
+JSON=$(yq r -j "$IDSREQFILE")
+
+# the return values from this function are ignored
+# we only use the echoed values
+run()
+{
+  local KEYCLOAK_ADDR=$1
+  local GETTOKEN="${KEYCLOAK_ADDR}/auth/realms/hello-world-authz/protocol/openid-connect/token"
```

Comment:
This entire path should be provided, the realm shouldn't be hardcoded. then we can just use `local GETTOKEN="${KEYCLOAK_ADDR}`